lib/Service/Sync/SyncService.php

```diff
@@ -103,9 +110,13 @@ public function syncMailbox(Account $account,
 		if ($partialOnly && !$mailbox->isCached()) {
 			throw MailboxNotCachedException::from($mailbox);
 		}
-
+		// construct imap client and perform initial connection
```

Comment:
```suggestion
```

https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/#h2-b7593c698a940